.circleci/config.yml

```diff
@@ -116,7 +116,7 @@ jobs:
       - run:
           name: evaluate [skipTests] condition
           command: |
-            echo "export COMMIT_MESSAGE=\"$(git log --format=oneline -n 1 $CIRCLE_SHA1)\"" >> $BASH_ENV
+            echo "export COMMIT_MESSAGE=\"$(git log --format=oneline -n 1 $CIRCLE_SHA1 | sed 's/\"/\\\"/g')\"" >> $BASH_ENV
```

Comment:
I think this fix will break if the commit message contains a `$`, because we'll end up sourcing a line that looks something like:
```
export COMMIT_MESSAGE="this is a $test"
```
which will try to substitute the value of the `test` variable.  Of course, a commit message like that isn't likely, until we change the code to be vulnerable, then it'll happen the next day, lol.

Could we directly set `COMMIT_MESSAGE`, without appending it to the BASH_ENV file?  Something like
```
export COMMIT_MESSAGE="$(git log --format=oneline -n 1 $CIRCLE_SHA1)"
```